contracts/badger-bridge/BadgerBridgeAdapter.sol

```diff
@@ -154,7 +167,7 @@ contract BadgerBridgeAdapter is OwnableUpgradeable, ReentrancyGuardUpgradeable {
         bytes calldata _btcDestination,
         uint256 _amount
     ) external nonReentrant {
-        require(_token == address(renBTC) || _token == address(wBTC), "invalid token address");
+        require(_token == address(renBTC) || _token == address(wBTC) || _token == address(ibBTC), "invalid token address");
```

Comment:
Let's just keep the `_token` value restricted to wbtc or renbtc (this is the token we have after unwrapping tokens from ibbtc -> vaults -> _token).